### PR TITLE
Spotfix: ⌗114442 Ensure Tooltip Shows Correct Label

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**⌗114442**](http://central.tri.be/issues/114442)

**Code Review Note:** No readme changes are required here.

### CODE REVIEW

File: src/Tribe/Google/Maps_API_Key.php [hunk 1]
```diff
@@ -50,15 +50,12 @@ public static function instance() {
 	public function filter_tribe_addons_tab_fields( array $addon_fields ) {
 
 		$tooltip = sprintf(
-			__( '<p><strong>You are using a custom Google Maps API key.</strong></p><br><p>%1$s to learn more about using it with The Events Calendar.</p>', 'the-events-calendar' ),
+			__( '<p><strong>You are using a custom Google Maps API key.</strong></p><p>%1$s to learn more about using it with The Events Calendar.</p>', 'the-events-calendar' ),
```
Comment (jesseeproductions) [inline src/Tribe/Google/Maps_API_Key.php]: It is a bit of a pain as it was already here, but lets separate out the html tags from the translation. 

Something like this:

'<p><strong>%1$s</strong></p><p>%2$s%3$s</p>'
__( 'You are using a custom Google Maps API key.', 'tec' ),
__( 'to learn more about using it with The Events Calendar.', 'tec' ),

**Comment by ggwicz** [comment]
Added a docblock to a new method; can confirm this is now _truly_ ready for code review!

File: src/Tribe/Google/Maps_API_Key.php (hunk 1 above)
Comment (ggwicz) [reply src/Tribe/Google/Maps_API_Key.php]: fixed!

**Comment by ggwicz** [comment]
Brian flagged my misunderstanding of the reworking here, I've implemented the correct feedback now and **now** can confirm this is ready for another review 🔍 ☕️

**Pull request merged**